### PR TITLE
Feat: Added images

### PR DESCRIPTION
**Pull request by Rohanmond**

### CODE REVIEW

File: components/image/images.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+<body>
+        <div class="img-responsive-container">
+            <img class="img-responsive" src="https://picsum.photos/1000/600" alt="">
+        </div>
+        <div class="img-container">
+            <img class="img-semi-square" src="https://picsum.photos/300/300" alt="">
+        </div>
+        <div class="img-container">
+            <img class="img-circle" src="https://picsum.photos/600/600" alt="">
+        </div>
```
Comment (anuj-kumary) [inline components/image/images.html]: Always give alt (alternate) in image.

File: components/image/image.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+.img-responsive{
+    width: 100%;
+    object-position: top center;  
+    object-fit: cover; 
+}
+.img-semi-square{
+    width: 100%;
+    border-radius: 10%;
+    object-position: top center;  
+    object-fit: cover;
```
Comment (Aman-Jain1996) [inline components/image/image.css]: You can make ageneric class with any name and give those properties to that class and add it to evry image you add , , one more suggestion try to add images of different sizes too and provide classes for those to user

**Review by Aman-Jain1996 — APPROVED** [review]
Everyting looks good to go except the comments

**Review by PeeyushPrashant — APPROVED** [review]
looks good, just make the changes as suggested above

File: components/image/images.html (hunk 1 above)
Comment (Rohanmond) [reply components/image/images.html]: done Thank for your review

File: components/image/image.css (hunk 2 above)
Comment (Rohanmond) [reply components/image/image.css]: Thanks for the review. Done the first one but for the second one we can have any size of parent container and the images will cover the parent container so there is no use of giving different classes for sizes.

**Pull request merged**